utils.py

```diff
@@ -291,16 +321,18 @@ def convert_png_binary_to_data_url(content):
     Raises:
         Exception. The given binary string does not represent a PNG image.
     """
-    if imghdr.what(None, h=content) == 'png':
+    if imghdr.what(None, h=content) == 'png': # type: ignore[call-overload]
```

Comment:
Why is this ignore needed?

Reply:
We accept unicode(or Text) but imghdr.what(file, h) accept 'h' of type str. So earlier I added type ignore call overload as we should avoid using `str` in py2 with mypy type annotations. 
Now I have another way to do this, `content = cast(str, content)`. This casts content to be of type str. Here cast is a method of module typing.

Reply:
If this is a temporary solution, until py3 is introduced should we add a TODO?

Reply:
In python3, this will change to bytes I think. Or what should be its type in python3? . So the whole type annotation needs to change after that. Don't think TODO is required here.